app/models/user.rb

```diff
@@ -19,27 +19,22 @@ def self.from_cas(access_token)
     user
   end
 
-  def self.all_users
-    User.all.map(&:uid)
-  end
-
+  # Users that can be project sponsors
   def self.sponsor_users
```

Comment:
` self.sponsor_users` and ` self.manager_users` below now returns a list of User objects (rather than just their `uid`s) so that we can display both the user name and their netid